src/pretix/control/logdisplay.py

```diff
@@ -96,6 +96,7 @@ def pretixcontrol_logentry_display(sender: Event, logentry: LogEntry, **kwargs):
         'pretix.event.order.payment.changed': _('The payment method has been changed.'),
         'pretix.event.order.expire_warning_sent': _('An email has been sent with a warning that the order is about '
                                                     'to expire.'),
+        'pretix.event.order.mail_sent': _('The order received a custom email.'),
```

Comment:
Better: A custom email has been sent. An order is not something that can receive emails ;)